docs/opensafely-cli.md

```diff
@@ -125,3 +125,21 @@ To pull the most recent Docker images to your machine, run:
 ```bash
 opensafely pull
 ```
+
+### Running JupyterLab
+
+[Jupyter notebooks](https://jupyter.org/) are useful interactive
+environments for developing code.
+
+You can run JupyterLab to use Jupyter notebooks via the `opensafely`
+tool. This ensures that the Python code you write will work in the
+OpenSAFELY environment.
+
+To use Jupyter notebooks via `opensafely`:
+
+1. Run `opensafely jupyter` to start a JupyterLab server on your
+   computer.
+2. You will see long URLs starting `http://localhost…` and
```

Comment:
It should automatically open a brower

Reply:
I'll mention that as the default, as it seems like it should be the case.

(This isn't the behaviour for me, and @iaindillingham said [similar](https://github.com/opensafely/documentation/discussions/530). This is why I specified the copy-paste step explicitly.)